### ISSUE TITLE
DDPG with default hyper-paras doesn't work in mujoco swimmer-v2 env

### ISSUE THREAD

**Issue by SiyuanLee** [openai/baselines#690]
Hi all, I have run DDPG with default hyperparameters in mujoco swimmer-v2 environment, but the reward converges to a very low value, only 4 or 5, so the swimmer cannot swim at all. I did not change the code, and run with the script: python -m baselines.run --alg=ddpg --env=Swimmer-v2 --num_timesteps=1e6 . I don't know where is wrong. Thank you for your help.

**Comment by iswaverly** [comment]
Have you ever try HalfCheetah? I got problem with it either. #764

**Comment by pzhokhov** [comment]
I suspect the issue is with hyperparameters (it would be uninteresting if the default ones worked in all environments, would it not ? ;) @iswaverly @SiyuanLee if you find the hyperparameter setting that works, please post it here.

**Comment by Yeosangho** [comment]
I have same issue. DDPG default parameter doesn't training or training very slowly in mujoco enviroments that I tested (Halfcheetah, Walker2d)

**Comment by sayomakinwa** [comment]
I have a similar situation with DDPG on Humanoid-v2 environment; it doesn't converge. Suggestions will be highly appreciated

**Comment by QiXuanWang** [comment]
Actually I tried some other envs(MountainCarContinuous-v0, CartPole-v0, etc).  None of them give me positive returns.  While using another implementation, MountainCar give me positive return. Not sure if it's DDPG implementation issue or just hyper-parameter tuning issue...
